website/templates/emails/discussions_email_rejected.txt.mako

```diff
@@ -0,0 +1,19 @@
+Hello${' {}'.format(user.fullname) if user else ''},
+
+The email that you sent to ${target_address} was rejected, and was therefore not forwarded.
+
+%if reason == 'no_user':
```

Comment:
They could be put into a dict and inserted into `mail_params`, but otherwise no.

Tests are in `/tests/test_discussions.py:TestEmailRejections`, though they only test the call to `send_mail` that renders the template, not the rendered template itself.